### PR TITLE
Update definition of public/private for repo policy check

### PR DESCRIPTION
**Pull request by ChumpChief**
In #10028 I discovered that examples are considered public, which seems an incorrect labeling since we don't publish them to NPM (and I believe the purpose of this check is to ensure published packages don't depend on non-published packages).  This change would do the following:

1. Define "public" and "private" based on whether the packages publish to NPM
    * Public: `@fluidframework`, `@fluid-experimental`
    * Private: `@fluid-internal`, `@fluid-example`, `@fluid-tools`
2. Update the "private" flags in package.json files to match this definition
3. Convert the bubblebench packages to "example" rather than "experimental" -- they were always under an examples subdirectory, I think this was just an oversight.
4. Add a missing LICENSE file to property-query-service.

### CODE REVIEW

File: experimental/PropertyDDS/services/property-query-service/package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "@fluid-experimental/property-query-service",
   "version": "0.59.3000",
```
Comment (markfields) [inline experimental/PropertyDDS/services/property-query-service/package.json:3]: I don't know anything about this package - is it ok to publish it?

Comment (ChumpChief) [reply experimental/PropertyDDS/services/property-query-service/package.json:3]: Good question - I'll instead convert it to `@fluid-internal` perhaps

**Comment by ChumpChief** [comment]
Just realized I'll need to break this up for integration - will push a PR to only do the policy check update and hold the repo updates until we integrate that.

File: experimental/examples/bubblebench/baseline/package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
-  "name": "@fluid-experimental/bubblebench-baseline",
+  "name": "@fluid-example/bubblebench-baseline",
   "version": "0.59.3000",
+  "private": true,
```
Comment (markfields) [inline experimental/examples/bubblebench/baseline/package.json:4]: Are you sure this belongs in examples not experimental scope? I see a few hundred weekly downloads on npmjs (feels high...?), and it seems more about testing the tech not an example of how to build XYZ?  I don't actually have any context here just trying to get a sense.

https://www.npmjs.com/package/@fluid-experimental/bubblebench-sharedtree

File: tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts [hunk 3]
```diff
@@ -29,21 +29,27 @@ or logos must follow Microsoft's [Trademark & Brand Guidelines](https://www.micr
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 `;
 
+// Packages which do not publish on NPM
 function packageShouldBePrivate(name: string): boolean {
     // allow test packages to be packaged
```
Comment (markfields) [inline tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts:34]:
```suggestion
    // allow test packages to be published
```

File: tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts [hunk 4]
```diff
@@ -29,21 +29,27 @@ or logos must follow Microsoft's [Trademark & Brand Guidelines](https://www.micr
...
     if (name.startsWith("@fluid-internal/test-")) {
         return false;
     }
 
     return (
-        name === "root" || // minirepo roots
-        name.startsWith("@fluid-internal"));
+        name === "root" // minirepo roots
```
Comment (markfields) [inline tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts:40]:
```suggestion
        name === "root" // monirepo roots
```

My coining the term "minirepo" never really caught on :P

File: packages/tools/fetch-tool/package.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@fluid-tools/fetch-tool",
   "version": "0.59.3000",
+  "private": true,
```
Comment (markfields) [inline packages/tools/fetch-tool/package.json:4]: Feels like a potential loss not to publish some tool packages.  Can we remove this namespace from both lists in the policy check so they can be classified on a case by case basis?

Comment (markfields) [reply packages/tools/fetch-tool/package.json:4]: Like you said, layer check should protect us here.  No layer says they may depend on Tools layer (If I'm reading layerInfo.json properly... might be good to test it out real quick)

File: experimental/examples/bubblebench/baseline/package.json (hunk 2 above)
Comment (ChumpChief) [reply experimental/examples/bubblebench/baseline/package.json:4]: I don't believe the content needs to be published for any particular reason (i.e. I don't think we'd advise customers to take a dependency on the package for any reason), and the content looks like a demo to me.  @DLehenbauer looks like the likely owner and might have a position.

File: packages/tools/fetch-tool/package.json (hunk 5 above)
Comment (ChumpChief) [reply packages/tools/fetch-tool/package.json:4]: Oh yikes, we didn't use to publish `@fluid-tools` AFAIK, not sure when that started.  I don't think we've audited those packages really, so not sure if they're really in a state where we should feel comfortable with external customers using them...  In particular I don't think we have any kind of support plan for them if a customer hits an issue necessarily.

File: tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts (hunk 4 above)
Comment (ChumpChief) [reply tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts:40]: monorepo?

File: packages/tools/fetch-tool/package.json (hunk 5 above)
Comment (markfields) [reply packages/tools/fetch-tool/package.json:4]: > Oh yikes, we didn't use to publish @fluid-tools AFAIK

Not following...

Comment (ChumpChief) [reply packages/tools/fetch-tool/package.json:4]: Followed up offline with Mark - I'll leave the `@fluid-tools` alone for now, and make a proposal as to what we should do with them.

**Comment by ChumpChief** [comment]
Closing in favor of #10065.

**Pull request closed without merging**